http/session_handler.go

```diff
@@ -176,9 +176,10 @@ func decodeCookieSession(ctx context.Context, r *http.Request) (string, error) {
 // SetCookieSession adds a cookie for the session to an http request
 func SetCookieSession(key string, r *http.Request) {
 	c := &http.Cookie{
-		Name:   cookieSessionName,
-		Value:  key,
-		Secure: true,
+		Name:     cookieSessionName,
+		Value:    key,
+		Secure:   true,
+		SameSite: 3, // SameSiteStrictMode
```

Comment:
Rather than using the number, use the self-documenting variable.
```suggestion
		SameSite: http.SameSiteStrictMode,
```